clang/unittests/Format/FormatTest.cpp

```diff
@@ -16671,9 +16667,8 @@ TEST_F(FormatTest, ConfigurableSpaceBeforeParens) {
                SpaceFuncDecl);
   verifyFormat("int f () throw(Deprecated);", SpaceFuncDecl);
   verifyFormat("typedef void (*cb)(int);", SpaceFuncDecl);
-  // FIXME these tests regressed behaviour.
-  // verifyFormat("T A::operator() ();", SpaceFuncDecl);
-  // verifyFormat("X A::operator++ (T);", SpaceFuncDecl);
+  verifyFormat("T A::operator()();", SpaceFuncDecl);
+  verifyFormat("X A::operator++(T);", SpaceFuncDecl);
```

Comment:
Why are the spaces removed?

Reply:
[D114696](https://reviews.llvm.org/D114696) reverted the behavior for function declaration and definition, to have no space before the parentheses.

Reply:
It seems that D114696 restored the space in `operator` function declarations and definitions.